db/access.php

```diff
@@ -33,9 +33,22 @@
         'contextlevel' => CONTEXT_COURSE,
         'archetypes' => [
             'editingteacher' => CAP_ALLOW,
-            'manager' => CAP_ALLOW
+            'manager' => CAP_ALLOW,
         ],
-        'clonepermissionsfrom' => 'moodle/course:manageactivities'
+        'clonepermissionsfrom' => 'moodle/course:manageactivities',
+    ],
+
+    // See all CMS types in the activity list.
+    'mod/cms:seeall' => [
+        'riskbitmask' => RISK_XSS,
+
+        'captype' => 'write',
+        'contextlevel' => CONTEXT_COURSE,
+        'archetypes' => [
+            'editingteacher' => CAP_ALLOW,
```

Comment:
I think by default the setting should be set to `isvisible = 1` but the cap should be `NOT_SET`

Reply:
Marcus, can you clarify what you mean by ```isvisible = 1```. Do you mean the database value?

Reply:
done